app/views/exploreWrapper.scala.html

```diff
@@ -0,0 +1,11 @@
+@import models.user.User
+@(title: String, user: Option[User] = None)(implicit lang: Lang)
+
+@main(title, Some("/explore")) {
+    @navbar(user, Some("/explore"))
+    <link rel="stylesheet" href='@routes.Assets.at("stylesheets/wrapperFrame.css")'/>
+    
+    <iframe id="wrapperFrame"></iframe>
```

Comment:
Should the ID be specific to the explore page (like "explore-wrapper-frame"), or is something like "wrapper-frame" preferred because it will be reusable?

Reply:
`wrapper-frame` is preferred because it'll be reusable. Fixed it to be `wrapper-frame` instead of `wrapperFrame` to match with the convention in the other element ids.